logger/logger.go

```diff
@@ -2,30 +2,54 @@ package logger
 
 import (
 	"os"
+	"strings"
 
 	"go.uber.org/zap"
 	"go.uber.org/zap/zapcore"
 )
 
-var Instance = zap.New(
-	zapcore.NewCore(
-		zapcore.NewConsoleEncoder(zapcore.EncoderConfig{
-			// TimeKey:        "ts",
-			LevelKey:       "level",
-			NameKey:        "logger",
-			CallerKey:      "caller",
-			MessageKey:     "message",
-			StacktraceKey:  "stacktrace",
-			LineEnding:     zapcore.DefaultLineEnding,
-			EncodeLevel:    zapcore.LowercaseLevelEncoder,
-			EncodeTime:     zapcore.ISO8601TimeEncoder,
-			EncodeDuration: zapcore.SecondsDurationEncoder,
-			EncodeCaller:   zapcore.ShortCallerEncoder,
-		}),
-		zapcore.AddSync(os.Stdout),
-		zap.NewAtomicLevelAt(zap.InfoLevel),
-	),
-).Sugar().Named("fd")
+var Instance *zap.SugaredLogger
+var Level zap.AtomicLevel
+
+func init() {
+	level := zap.InfoLevel
+	switch strings.ToLower(os.Getenv("LOG_LEVEL")) {
```

Comment:
I still think the initial debug level should be passed through command line options, not environment variables. Would love to hear @vano144 @ansakharov 's opinions on this.   
If we decide however to stick to your option, we should document it well. Now this just looks like some undocumented magic only file.d developers are aware of.

Reply:
Done
Added to documentation